cli/commands/pack.go

```diff
@@ -163,6 +166,10 @@ func runPackCommand(cmd *cobra.Command, args []string) error {
 	ctx.Project.Path = cmd.Flags().Arg(1)
 	ctx.Cli.CartridgeTmpDir = os.Getenv(cartridgeTmpDirEnv)
 
+	if err := setDefaultValue(cmd.Flags(), "net-msg-max", strconv.Itoa(defaultNetMsgMax)); err != nil {
+		return project.InternalError("Failed to set default net_msg_max value: %s", err)
```

Comment:
Is this InternalError really needed here?

Reply:
I did the same as `log` https://github.com/tarantool/cartridge-cli/blob/master/cli/commands/log.go#L53

Reply:
This is needed for logs to differ when 0 is specified by the user or nothing is specified. Seems that here you can do it simpler. Just use `0` default value for `net-msg-max` to indicate that this parameter shouldn't be set at all.